server/protocols/jmap-rfc-8621/doc/specs/spec/mail/message.mdown

```diff
@@ -363,7 +363,7 @@ An **EmailBodyPart** object has the following properties:
 In addition, the client may request/send EmailBodyPart properties representing individual header fields, following the same syntax and semantics as for the Email object, e.g., `header:Content-Type`.
 
 <aside class="warning">
-// I have some confused: this part we already supported
+ We support it for Email, but not yet for EmailBodyPart
```

Comment:
We support fetching specific headers for the Email entity, but this is not yet implemented for the EmailBodyPart entity.